### PR TITLE
Subscriber Stats reports origin and edge server

### PR DESCRIPTION
**Pull request by eauge**
Subscriber Stats gets for each subscriber the edge server and the origin server

### CODE REVIEW

**Comment by eauge** [comment]
@aullman could you please review?

**Comment by aiham** [comment]
@eauge Can you make the tests pass? And add tests to cover the new code that you've added? Thanks.

File: src/js/subscriber-stats.js [hunk 1]
```diff
@@ -102,7 +118,10 @@ angular.module('opentok-meet').directive('subscriberStats', ['OTSession', 'Stats
         '</div><div ng-show="stats.video">' +
         'Video Packet Loss: {{stats.videoPacketLoss}}%<br/>' +
         'Video Bitrate: {{stats.videoBitrate}} kbps' +
-        '</div></div>',
+        '</div><div ng-show="stats.info">' +
+        'Origin server: {{stats.info.originServer}}<br/>' +
+        'Edge server: {{stats.info.edgeServer}}' +
+        '</div>',
```
Comment (aiham) [inline src/js/subscriber-stats.js]: You're missing a `</div>` here to close `<div class="show-stats-info" ng-show="showStats">` from above.

**Comment by eauge** [comment]
the unit tests should be passing, at least they are passing for me locally

**Comment by aullman** [comment]
@eauge ok, the tests are passing now. But I noticed that the edge and origin servers appear and disappear in the stats repeatedly. Also you should add the Origin and edge server sections to the stats integration tests here:
https://github.com/opentok/opentok-meet/blob/master/tests/e2e/scenarios.js#L690

You just need to update that RegExp to include it.

**Comment by eauge** [comment]
@aullman I've updated the PR with the regexp. Also, the origin and edge servers should not be appearing and disappearing anymore, the http request is done only once. Let me know if you see any other issues.

Thanks!

**Comment by eauge** [comment]
retest this please

**Comment by aullman** [comment]
LGTM

**Pull request merged**